### PR TITLE
refactoring into a reusable skeleton

### PR DESCRIPTION
**Pull request by brodavi**
moved assets folder up, created content folder, added config file, updated package.json, gulp-postcss as dependency

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules
+dist
```
Comment (eethann) [inline .gitignore]: missing newline :P

File: config-dev.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  port: 1337,
+  awsRegion: 'us-east-1',
+  awsBucket: 'awesomeactionpage.com',
+  awsCache: '/tmp/actionpage.cache',
+  cloudFrontDistID: 'ABCD1234',
+  title: 'Action-Page Title',
+  metatitle: 'Action-Page',
+  metaurl: 'http://awesomeactionpage.com/',
+  metaappid: '1234567890',
```
Comment (eethann) [inline config-dev.js]: is this facebook app id? might be good to clarify the key if so.

File: config-dev.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  port: 1337,
+  awsRegion: 'us-east-1',
+  awsBucket: 'awesomeactionpage.com',
+  awsCache: '/tmp/actionpage.cache',
+  cloudFrontDistID: 'ABCD1234',
+  title: 'Action-Page Title',
+  metatitle: 'Action-Page',
+  metaurl: 'http://awesomeactionpage.com/',
```
Comment (eethann) [inline config-dev.js]: default to https 😄

File: content/bumper-sticker.pug [hunk 4]
```diff
@@ -0,0 +1,6 @@
+h2 For your car
```
Comment (eethann) [inline content/bumper-sticker.pug]: this could probably be generally expanded to "merchandise.pug" or similar.

File: content/index.pug [hunk 5]
```diff
@@ -0,0 +1,37 @@
+html
+	head
+		title #{title}
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		meta(property="og:title" content=metatitle)
+		meta(property="og:url" content=metaurl)
+		meta(property="fb:app_id" content=metaappid)
+		meta(property="og:description" content=metaappdescription)
```
Comment (eethann) [inline content/index.pug]: can we use a description that fully communicates the meta tag property name? e.g. `meta["property:name"]` perhaps?

File: content/twitter-share.pug [hunk 6]
```diff
@@ -0,0 +1 @@
+a#button__twitter-share.button.button--blue(href="https://twitter.com/intent/tweet?url=http%3A%2F%2Fnosunoco.com&text=DAPL%20profits%20and%20violence%20at%20Standing%20Rock%20won%27t%20fuel%20my%20holiday%20travel.%20Boycott%20Sunoco!&hashtags=nosunoco,nodapl&related=nosunoco", onclick="doTrackTwitterShare()") Share on Twitter
```
Comment (eethann) [inline content/twitter-share.pug]: This should be updated to use page URL from config file, etc.

**Review by eethann — COMMENTED** [review]
Good start! I personally like having all the src files in a single directory (keeps them nicely tidy from deploy stuff, local scripts, etc.). But if you like this better, go with it!

File: .gitignore (hunk 1 above)
Comment (brodavi) [reply .gitignore]: done!

**Comment by brodavi** [comment]
@eethann made more stuff customizable via config.js. How should I approach the customizable fonts/colors issue though? Is there some sweet spot you have in mind between full CSS customizability and set 'themes'?

File: config-dev.js [hunk 7]
```diff
@@ -5,13 +5,31 @@ module.exports = {
...
-  metaimage: 'https://awesomeactionpage.com/actionpage.png',
-  metatwittersite: '@awesomeactionpage',
-  mainlogo: 'action-page-logo.svg',
-  textTwitterMessage: encodeURIComponent("A boilerplate for creating compelling, super-fast one-page action landings for Action Network campaigns using just HTML, CSS and AWS."),
-  fbpagetitle: 'Action Page'
+  mainlogo: 'nosunoco-logo.svg',
+  actionNetwork: {
+    id: 'boycott-sunoco'
+  },
+  meta: {
```
Comment (eethann) [inline config-dev.js:12]: NICE!

**Review by eethann — COMMENTED** [review]
My instinct is to go with making a system that's easy to totally retheme via customized CSS, but could also work with a few variable changes for colors and a custom logo. Does that help?

**Pull request closed without merging**